service/history/api/recordactivitytaskstarted/api.go

```diff
@@ -48,6 +49,17 @@ import (
 	"go.temporal.io/server/service/history/workflow"
 )
 
+type RejectCode int32
+
+const (
+	reject_code_undefined       RejectCode = 0
+	activity_accepted           RejectCode = 1
+	rejected_stamp              RejectCode = 2
```

Comment:
```suggestion
	rejectedStampMismatch              RejectCode = 2
```

Should all be camelCase : https://go.dev/doc/effective_go#mixed-caps

Reply:
Another good convention is to start all of them with `rejectedCode`. I.e. `rejectedCodeStampMismatch`, `rejectedCodePaused`, etc. It helps intellisense also.

Reply:
renamed